src/app/submissions/submissions.component.html

```diff
@@ -22,6 +22,10 @@
         <mat-header-cell *matHeaderCellDef i18n>Status</mat-header-cell>
         <mat-cell *matCellDef="let element">{{element.status}}</mat-cell>
       </ng-container>
+      <ng-container matColumnDef="user">
+        <mat-header-cell *matHeaderCellDef i18n>Submitted by</mat-header-cell>
+        <mat-cell *matCellDef="let element">{{element.user.name}}</mat-cell>
```

Comment:
could you add `?` operator. sometimes while recording survey user might not fill details